Resources/Public/JavaScript/ContextMenuActions.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+define(["require", "jquery", "TYPO3/CMS/Backend/Viewport"], function (e, r, s) {"use strict";return function () {return e.pagesNewTree = function () {var n = r(this).data("page-new-tree-url");n && s.ContentContainer.setUrl(n)}, e}()});
```

Comment:
It defines the callback action we refer to in ItemsProvider

Reply:
Please copy the non-minified version from core then (and optionally estabilish a build task to minify it here). If it's identical to the core's version, why not load it from there?

Reply:
Hi Markus. It's not the same as the core version. It's written specifically for this context menu action. I de-obfuscated it and removed the copy pasted copyright block.

I don't think minification is needed since it's just a small file. Please add a build task if you wish.

Reply:
Fine with me. Thanks.